joomla/joomla-cms
# Postinstall for the case that the Update Cache time is eq 0

Pull Request for Issue #9789 
### Summary of Changes

This PR implemts a postinstall message that inform the users that they will never get mails with that setting
### Testing Instructions
- go to com_postinstall
- hide all messages
- check you com_installer config that the cache timout is != 0
- please make sure the update mail plugin is enabled
- apply this patch
- go to you database and fire the SQL command in the update file with you database prefix.
- go to com_postinstall
- see no messages
- go to the com_instaler settings again and set the cache timout to `0`
- got to com_postinstall
- see the message (please double check typos and suggest changes to the strings!)
- hit the action button
- see the message is gone.
- got to com_installer options again
- see the cache timout is 6 now
- set it back to 0
- disable the update mail plugin
- go to com_installer
- check that the message is not there
### Documentation Changes Required

None
